packages/mobile/src/app/selectors.test.ts

```diff
@@ -0,0 +1,41 @@
+import { verificationPossibleSelector } from 'src/app/selectors'
+import { getMockStoreData } from 'test/utils'
+import { mockE164Number, mockE164NumberPepper } from 'test/values'
+
+describe(verificationPossibleSelector, () => {
+  it('returns true', () => {
```

Comment:
nit: it's more helpful when the description gives more details about the expectations. 
for instance `it returns true when the number pepper is already cached or when the balance is sufficient to fetch it`.